Added CardPushDate plugin

dmorlitz opened the pull request:
Added a plugin which puts quick action buttons on task cards, including pushing due date out, view last comment, edit, close and move cards.

Commit:
Update plugins.json

Commit:
Update index.html

fguillot commented on plugins.json:
```diff
@@ -167,6 +167,18 @@
+        "description": "Put easy actions on task cards, searches and dashboard - including update card date, show comments, edit, close and move card",
+        "homepage": "https://github.com/dmorlitz/kanboard-CardPushDate",
+        "readme": "https://raw.githubusercontent.com/dmorlitz/kanboard-CardPushDate/master/README.md",
+        "download": "https://github.com/dmorlitz/kanboard-CardPushDate/archive/CardPushDate.zip",
```
Please, follow the instructions to build the archive: https://github.com/kanboard/website#how-to-add-a-new-plugin-to-the-list

Commit:
Update plugins.json

Built ZIP files according to instructions to enable auto-install of previously listed plugins - and added 1 new plugin

dmorlitz commented:
I believe I have rebuilt the necessary ZIP files based on the instructions you sent.  I have updated the ZIP files on my previous plugins as well as fixed the new plugin description in this pull request.

Please let me know if I missed something or made a mistake.

fguillot commented:
LGTM 👍
